docsource/modules150-160.rst

```diff
@@ -162,7 +162,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | google_recaptcha                                |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| google_spreadsheet                        |                      |                                                 |
+| |del| google_spreadsheet                        |Deprecated module     |Obsolete module, features completely removed.    |
```

Comment:
```suggestion
| |del| google_spreadsheet                        | Nothing to do        | Obsolete module                                 |
```
shorter

Reply:
This is not Nothing to do, as there's no replacement, so the text should specify that.

Reply:
![Selection_1488](https://github.com/OCA/OpenUpgrade/assets/25005517/be3f57c8-afee-4116-8318-bfa6108718b1)

This are the only valid options for the first column.

Reply:
Not really. If you put any other value, it's not parsed, but anyway, I prefer empty, and put the explanation. `Feature removed. Search for alternatives` or similar in the next column.

Reply:
Ok, that's fine for me. In https://github.com/OCA/OpenUpgrade/blob/14.0/docsource/modules130-140.rst, we can see that the first column for `web_diagram` was left empty, and as many other similar cases in other versions.

Reply:
Ok, thank you both, i'll go with empty and put the explanation then

Reply:
> ![Selection_1488](https://user-images.githubusercontent.com/25005517/251484348-be3f57c8-afee-4116-8318-bfa6108718b1.png)
> 
> This are the only valid options for the first column.

May i ask, where you get this, i don't know it does exist?